src/db/Timesheet.ts

```diff
@@ -1,12 +1,59 @@
 import { z } from "zod";
 
+/**
+ * Represents the database schema for a comment
+ */
+export const CommentSchema = z.object({
+  Type: z.string(),
+  AuthorUUID: z.string(),
+  Timestamp: z.number(),
```

Comment:
⛏️ Kind of want to keep the wording consistent with the other schemas, maybe `Date` (or even better, `DateTime`)?